yaki_mobile/lib/presentation/displaydata/declaration_card_content.dart

```diff
@@ -0,0 +1,38 @@
+// values are the json object's keys from assets/translations/*.json files
+const String remote = 'Remote';
+const String onSite = 'On site';
+const String vacations = 'Vacation';
+const String other = 'Other';
+
+final List statusCardsContent = [
+  {
+    'image': 'assets/images/remote.svg',
+    'text': remote,
+  },
+  {
+    'image': 'assets/images/onsite.svg',
+    'text': onSite,
+  },
+  {
+    'image': 'assets/images/vacation.svg',
+    'text': vacations,
+  },
+  {
+    'image': 'assets/images/dots.svg',
+    'text': other,
+  },
+];
+
+final Map<String, String> statusImage = {
+  remote: 'assets/images/remote.svg',
+  onSite: 'assets/images/onsite.svg',
+  vacations: 'assets/images/vacation.svg',
+  other: 'assets/images/dots.svg',
+};
+
+final Map<String, String> statusMessage = {
+  remote: "StatusRemote",
+  onSite: "StatusOnSite",
+  vacations: "StatusVacation",
+  other: "StatusOther",
+};
```

Comment:
all of this stuff can be just replaced by
```suggestion

String getStatusImage(Status status) => 'assets/images/${status}.svg';

String getStatusTranslationKey(Status status) => 'status_${status}'
```

Reply:
need to list to map  and display all cards, would need rework all status message to use the suggested function, will rework this once build work. Keeping this